### PR TITLE
UI tweaks

### PR DESCRIPTION
**Pull request by dirkgr**

### CODE REVIEW

File: tango/server/index.js [hunk 1]
```diff
@@ -6,13 +6,38 @@ fetch("/api/runlist", {
...
+    trHead.appendChild(thName);
+    thName.className='header__item';
+    thName.innerHTML = "Run Name";
+    const thStartDate = document.createElement("th");
+    trHead.appendChild(thStartDate);
+    thStartDate.className='header__item';
+    thStartDate.innerHTML = "Start Date";
+    out
+      .map(v => { return { name:v.name, startDate:dayjs(v.start_date) } })
+      .sort((a, b) => b.startDate-a.startDate)
```
Comment (jonborchardt) [inline tango/server/index.js:25]: sorting is here @dirkgr

File: tango/server/report.js [hunk 2]
```diff
@@ -161,19 +161,33 @@ const formatText = (label, value) => {
 
 // convert the data to a node
 const getTable = (data) => {
+  let mainBgColor = colors.B10;
+  switch (data.state) {
```
Comment (jonborchardt) [inline tango/server/report.js:165]: colors are here @dirkgr

File: tango/server/report.js [hunk 3]
```diff
@@ -261,7 +275,7 @@ const render = () => {
           }" fontsize="12" fontname="Helvetica"];
           edge [penwidth=2.0 color="${colors.N9}"]
           concentrate=True;
-          rankdir=TB;
+          rankdir=LR;
```
Comment (jonborchardt) [inline tango/server/report.js:278]: horizontal vs vertical is here @dirkgr

**Review by jonborchardt — APPROVED** [review]

**Comment by dirkgr** [comment]
@epwalsh, can you review this for the server changes, especially the `Run` class?

**Review by epwalsh — COMMENTED** [review]
I think I found a bug. The server doesn't show me any steps even though I know there should be:
image

To reproduce:

```bash
tango run test_fixtures/experiment/multiprocessing.jsonnet -i test_fixtures.package -d /tmp/tango
tango server -d /tmp/tango
```

**Comment by dirkgr** [comment]
@epwalsh try clearing the cache. There is a problem where even hitting F5 doesn't refresh the cache for the .js file.

**Comment by epwalsh** [comment]
Even after clearing the cache and/or opening in a private window I still have this issue, at least with Firefox. I did see the step for my run when I used Safari though:

image

But the box for that one step is very small. I don't know if that's an easy fix or not.. I've also noticed that before with Firefox.

**Review by epwalsh — COMMENTED** [review]
I'm happy with the `Run` class

**Pull request merged**